spec/support/table_methods.rb

```diff
@@ -0,0 +1,19 @@
+module TableMethods
+  def columns(table_name)
+    ActiveRecord::Base.connection.columns(table_name)
+  end
+
+  def unique_indexes_from(table_name)
+    indexes = indexes_from(:comments)
+    indexes.select(&:unique).map(&:name)
+  end
+
+  def indexes_from(table_name)
+    ActiveRecord::Base.connection.indexes(:comments)
+  end
+
+  def tables
+    tables = ActiveRecord::Base.connection.select_all('SHOW TABLES')
+    tables.flat_map { |table| table.values }
```

Comment:
It's used instead of `map`+`flatten`.  We have a result set like:

`[{"Tables_in_teambox_development"=>"activities"}, {"Tables_in_teambox_development"=>"addresses"}, {"Tables_in_teambox_developme...`

and with `flat_map`

`["activities", "addresses", "affiliates",...`